### DISCUSSION TITLE
ProductFamily instead of ProductGroup

### DISCUSSION THREAD

**Discussion by olivier5741** [schemaorg/schemaorg#3029] [Ideas]
Hi there, I think that ProductFamily would be a better wording for ProductGroup (https://schema.org/ProductGroup). 

Used in AtroPIM for instance (I'm not related to them :p) : 

* https://atropim.com/features -> search for "family"
* https://demo.atropim.com/#ProductFamily (login: admin, password: admin)

Cheers ;),
Olivier

**Comment by thadguidry** [comment]
We won't change the name, I think because of the thousands of sites already using it.  It's appropriately named for the context in its description.  However, you might be interested in ProductCollection and perhaps CategoryCode.
